dmscripts/helpers/env_helpers.py

```diff
@@ -1,9 +1,9 @@
 def get_api_endpoint_from_stage(stage, app='api'):
-    """Return the full URL of given API or Search API environment.
+    """Return the full URL of given API, Search API or web URL environment.
 
     :param stage: environment name. Can be one of 'preview', 'staging',
                   'production' or 'dev' (aliases: 'local', 'development').
-    :param app: should be either 'api' or 'search-api'
+    :param app: should be either 'api', 'search-api' or 'www'
```

Comment:
How about splitting the difference and having a new `get_web_url_from_stage` function in `env_helpers`? As we're now going to append the `utm` param in the code, it makes sense to me to generate the whole URL first then pass it to the template.